.github/CODEOWNERS

```diff
@@ -3,5 +3,32 @@
 ################
 
 # Git Hub bot rules
-/.github/CODEOWNERS  @jsquire @ronniegeraghty
-/.github/fabricbot.json  @jsquire @ronniegeraghty
+/.github/CODEOWNERS                           @jsquire @ronniegeraghty
+/.github/fabricbot.json                       @jsquire @ronniegeraghty
+
+#####################
+# Code Generation
+#####################
+
+# Catch all
+*                                              @AlexanderSher @lirenhe @chunyu3 
+
+# Data Plane Reviewers
+/samples/AppConfiguration/                     @annelo-msft
+/samples/Azure.AI.*/                           @annelo-msft
+/samples/Azure.Analytics.*/                    @annelo-msft
+/samples/Azure.Storage.*/                      @annelo-msft
+/samples/Cognitive*/                           @annelo-msft
+/src/assets/Generator.Shared/                  @annelo-msft
+/src/AutoRest.CSharp/DataPlane/                @annelo-msft
+/src/AutoRest.CSharp/LowLevel/                 @annelo-msft
+/test/AutoRest.TestServer.Tests/               @annelo-msft
+/test/AutoRest.TestServerLowLevel.Tests/       @annelo-msft
+
+# Management CSharp CodeReviewers
+/samples/Azure.Management.Storage/             @allenjzhang @m-nash
+/samples/Azure.Network.Management.Interface/   @allenjzhang @m-nash
+/samples/Azure.ResourceManager.*/              @allenjzhang @m-nash
+/src/AutoRest.CSharp/Mgmt/                     @allenjzhang @m-nash
+/src/AutoRest.CSharp/Common/                   @allenjzhang @m-nash
+/test/AutoRest.TestServer.Tests/Mgmt/          @allenjzhang @m-nash
```

Comment:
@ArthurMa1978, @allenjzhang, @m-nash:  I did not update the management owners.   Please let me know if you'd like me to do so.